_data/blogs.yml

```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: Screenshot(21).jpg
```

Comment:
Please upload this image

Reply:
I am not able to upload the image.Please Help!!

Reply:
Please tell me how do I do it.

Reply:
@hshafaq7 Kindly add the image to the following folder and update the commit 👍 tell us if you are facing any issue.

Reply:
Which Folder